Kitodo/src/main/java/org/kitodo/production/helper/VariableReplacer.java

```diff
@@ -387,6 +387,24 @@ private String determineReplacementForMetadata(Matcher variableFinder) {
         }
     }
 
+    private String determinateReplacementForAll(Matcher variableFinder, String dollarSignIfToKeep) {
+        List<LogicalDivision> allChildren = workpiece.getLogicalStructure().getChildren();
+        String allFirstchildValue = null;
+        if (!allChildren.isEmpty()) {
+            allFirstchildValue = MetadataEditor.getMetadataValue(allChildren.get(0), variableFinder.group(5));
+            if (Objects.isNull(allFirstchildValue)) {
+                allFirstchildValue = determineReplacementForTopstruct(variableFinder, dollarSignIfToKeep);
+            }
+            if (Objects.isNull(allFirstchildValue) || StringUtils.isEmpty(allFirstchildValue)) {
```

Comment:
[`StringUtils.isEmpty()`](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isEmpty-java.lang.CharSequence-) implies a `null` check, so you can remove the `Objects.isNull()` check